app/build.gradle.kts

```diff
@@ -137,7 +139,7 @@ android.apply {
 }
 
 kotlin {
-    jvmToolchain(11)
+    jvmToolchain(17)
```

Comment:
jestli ten suppress na magic number je jen kvuli tomuhle, nejde to nahradit treba `JavaVersion.VERSION_17.majorVersion.toInt()` ?

Reply:
jj je to tam jen kvůli tomu upravím díky